docs/directives/needarch.rst

```diff
@@ -41,14 +39,65 @@ Jinja context
 
 The following Jinja functions are **only available** for :ref:`needarch`. 
 
+
+.. _needarch_jinja_need:
+
+need()
+~~~~~~
+
+.. versionadded:: 1.0.3
+
+The `need()` function provides you the need information the :ref:`needarch` / :ref:`needuml` is embedded in.
```

Comment:
need() should not be accessible in needuml, as needuml can be used outside a need.

So the same code would behave differently, if used inside a need or not.
I personally don't like this :/

Reply:
Implemented as requested. Even covered with tests in needarch (to support need()) and needuml (to get exception in case of use need()).